code-challenges.test.js

```diff
@@ -22,41 +22,135 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // b) Create the function that makes the test pass.
 
-
+// Canned code below for function tests
+// describe("functionName", () => {
+//   it("describes what my test and function does", () => {
+//     expect(functionName("pass any params needed")).toEqual("Expect output")
+//   })
+// })
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
+// Psuedocode:
+
+// 1. Create/declare a function called divisibleByThree
+// 2. Pass a parameter called number. This number will be used in the function
+// 3. Use an if statement if the number modulo 3 is strictly equal to zero
+    // 3a. Use a return statement that the number is divisible by three
+// 4. Otherwise, use an else statement
+    // 4a. Then use a return statement that the number is not divisible by three
+
+
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("divisibleByThree", () => {
+  it("takes a number as an argument and decides if the number is evenly divisble by three or not.", () => {
+    expect(divisibleByThree(15)).toEqual(15 + " is divisible by three")
+    expect(divisibleByThree(0)).toEqual(0 + " is divisible by three")
+    expect(divisibleByThree(-7)).toEqual(-7 + " is not divisible by three")
```

Comment:
I would recommend avoiding using any type of concatenation or interpolation in your test.  The test is meant to just compare an output of your function to a predefined expected output.